src/Repository/ProgrammeRepository.php

```diff
@@ -44,4 +50,24 @@ public function findAllFiltered(
 
         return $query->getQuery()->getResult();
     }
+
+    public function countBusyProgrammes(): array
+    {
+        $conn = $this->getEntityManager()->getConnection();
+
+        $sql = '
+            SELECT
```

Comment:
This query returns all the hours from all days, with their number of participants.

the requirement was to only show the busiest hour from each day(not all the hours like your query does)